ext/standard/basic_functions.stub.php

```diff
@@ -768,6 +768,140 @@ function scandir(string $directory, int $sorting_order = 0, $context = UNKNOWN):
 function glob(string $pattern, int $flags = 0): array|false {}
 #endif
 
+/* exec.c */
+
+function exec(string $command, &$output = null, &$result_code = null): string|false {}
```

Comment:
It seems to me like the "Unable to fork" error condition may return null. We should make that return false...